src/Autonomous.cpp

```diff
@@ -142,34 +135,32 @@ void Autonomous::autonomyIter()
 	{
 		std::cout << "arrived at gate" << std::endl;
 		std::cout << "x: " << pose(0) << " y: " << pose(1) << " theta: " << pose(2)
-				  << std::endl;
+					<< std::endl;
 		landmarkFilter.reset(); // clear the cached data points
 		setCmdVel(0, 0);
 	}
 	else
 	{
-		PointXY driveTarget = getTarget();
-		if (dist(driveTarget, point_tToPointXY(pose)) < 25)
```

Comment:
I removed this check. If we see the AR tag from farther away than 25 meters, we still might as well use that as our navigation target.